src/helpers/Query.ts

```
@@ -183,6 +184,110 @@
+		if (!options.request || !options.schema) {
+			this.logger.error('[Query][buildFindManyOptionsFromRequest] Request or Schema not provided')
+			return
+		}
+
+		const searchRequest = new URLSearchParams(options.request)
+		const request = Object.fromEntries(searchRequest.entries())
+		
+		let sort
+
+		if(request['sort']) {
+
```

ESLint

[error] Replace `(request['sort'])·{⏎` with `·(request['sort'])·{`

(prettier/prettier)